recipes/qd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.3.17" %}
+
+package:
+  name: qd
+  version: {{ version }}
+
+source:
+  fn: qd-{{ version }}.tar.gz
+  url: http://crd.lbl.gov/~dhbailey/mpdist/qd-{{ version }}.tar.gz
+  sha256: c58e276f6fcf5f2f442c525f3de42ea00004734572b29c74028bbda0ad81096d
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+  run:
+    - libgcc
```

Comment:
4.8.5 I think, but yes they are different.

IIUC `libgcc` 5 is backwards compatible with `libgcc` 4. The biggest changes for `gcc` were for C++11 support, but they provided both the old and new symbols in the new `libgcc`.

So not something we should need to worry about.